### PR TITLE
q43

### PR DESCRIPTION
**Pull request by brar-navleen**
q43:
image

### CODE REVIEW

File: Javascript/q43.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      width: 300px;
+    }
+
+    .box {
+      width: 200px;
+    }
+  </style>
+
+  <script>
+    function myFunction() {
```
Comment (Mantaseus) [inline Javascript/q43.html:27]: The function name can be improved

File: Javascript/q43.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      const min = Number(document.querySelector("#demo0").value);
+      const max = Number(document.querySelector("#demo1").value);
+      const count = Number(document.querySelector("#demo2").value);
+      const arr = [];
+  
+      while(arr.length < count) {
+        arr.push(Math.round(Math.random() * (max - min + 1) + min ));
```
Comment (Mantaseus) [inline Javascript/q43.html:34]: I think this formula will give you random numbers between `min` and `max + 1` inclusive. The question is asking for a random number between `min` and `max` inclusive

File: Javascript/q43.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+</head>
+
+<body>
+  <div class="flex">
+    <div>Min limit</div>
+    <input id= "demo0" type="number" class="box" style="margin-left: 4px;"></input>
+    <div>Max limit</div>
+    <input id="demo1" type="number" class="box"></input>
+    <div>Count</div>
+    <input id="demo2" type="number" class="box" style="margin-left: 24px;"></input>
```
Comment (Mantaseus) [inline Javascript/q43.html:48]: The class ID names can be improved

File: Javascript/q43.html (hunk 1 above)
Comment (Navleenkaursandhu) [reply Javascript/q43.html:27]: DONE

**Pull request merged**